src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,88 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final float LOAD_FACTOR = 0.75f;
+    private static final int DEFAULT_LENGTH = 16;
+    private static final int LENGTH_MULTIPLIER = 2;
+    private int size;
+    private Node<K, V>[] data;
+
+    public MyHashMap() {
+        data = new Node[DEFAULT_LENGTH];
+    }
 
     @Override
     public void put(K key, V value) {
-
+        int hash = hash(key);
```

Comment:
```suggestion
        int index = getIndex(key);
```
давай зробимо так щоб метод повертав нам відразу індекс (позицію в масиві)